doc/adrs/053_add_support_for_form_descriptions_in_the_view_converter.adoc

```diff
@@ -0,0 +1,42 @@
+= ADR-053 - Add support for form descriptions in the view converter
+
+== Context
+
+The RepresentationDescriptions in the Vew DSL needs to be converted in the supported type `org.eclipse.sirius.components.representations.IRepresentationDescription`.
+For now, only `DiagramDescription` are converted in the `ViewConverter`.
+
+== Decision
+
+We need to create and add a `FormDescriptionConverter` to the `ViewConverter`.
+The `ViewConverter.convert()` method statically converts DiagramDescription:
+
+```            
+result = view.getDescriptions().stream()
+    .filter(org.eclipse.sirius.components.view.DiagramDescription.class::isInstance)
+    .map(org.eclipse.sirius.components.view.DiagramDescription.class::cast)
+    .map(viewDiagramDescription -> this.convertDiagramDescription(viewDiagramDescription, interpreter))
+    .collect(Collectors.toList());
+                         
+```
+
+We need to modify this logic to dispatch to the correct converter according to the `RepresentationDescription` concret type.
+
+The `FormDescriptionConverter` will create a new `org.eclipse.sirius.components.forms.description.FormDescription`. The DSL does not handle yet Pages and Groups. The converter will create a default page and group.
```

Comment:
Please do not extend it in an hardcoded way, just make it take some `IRepresentationDescriptionConverter` and provide a `FormDescriptionConverter` along the `DiagramDescriptionConverter`. A key part to make it support new kinds of representation descriptions is to allow it to transform representation descriptions that it does not know.

Reply:
OK, I will clarify this part.